packages/material-ui/src/Select/SelectInput.test.js

```diff
@@ -41,7 +42,7 @@ describe('<SelectInput />', () => {
 
   it('should render a correct top element', () => {
     const wrapper = shallow(<SelectInput {...defaultProps} />);
-    assert.strictEqual(wrapper.name(), 'div');
+    assert.strictEqual(wrapper.name(), 'Fragment');
```

Comment:
We might as well remove it. The test does not provide any value. Nobody cares what the "top element" is. We don't have any public API about what a "top element" is.

Reply:
Yeah.